src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1786,26 +1788,28 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector rawCerV;
+            ContextElementResponseVector rawCerV2;
```

Comment:
Ah, ok, I think I get the point...

Redeclaring was wrong here. Thus, remove it, avoid shadowing and let's use the same variable (free at this point of code) for this block of code. Makes sense?